### PR TITLE
support for multiple aggregation functions

### PR DESCRIPTION
**Pull request by jim-smith**

### CODE REVIEW

**Comment by jim-smith** [comment]
@rpreen also fixes side issue that asking for marginal totals broke the crosstab function because the value of margins was not being passed to. the creation of the masks

File: test/test_initial.py [hunk 1]
```diff
@@ -37,6 +37,28 @@ def test_crosstab_without_suppression(data):
...
+    )
+    output = acro.results.get_index(0)
+    correct_summary: str = (
+        "fail; threshold: 12 cells may need suppressing;"
+        " p-ratio: 2 cells may need suppressing; "
+        "nk-rule: 2 cells may need suppressing; "
+    )
+    assert (
+        output.summary == correct_summary
+    ), f"\n{output.summary}\n should be \n{output.summary}\n"
```
Comment (rpreen) [inline test/test_initial.py:55]: should be `correct_summary`?

Comment (rpreen) [reply test/test_initial.py:55]: pytest output tells you the values though anyway if it fails, so not sure why it's needed after the assert?

Comment (jim-smith) [reply test/test_initial.py:55]: 'cos running pytest in my laptop gives me a fairly messy output and crops long lines.
That's why I used an f-string with newlines in it so it's easier to look at them aligned the same.
Not a big deal.

Comment (jim-smith) [reply test/test_initial.py:55]: but yes you are right about the f-string. Will change

Comment (jim-smith) [reply test/test_initial.py:55]: @rpreen don't understand GitHub :( I've pushed a new commit, and it shows below, and if I look. at the files through the code viewer. assume this is ok

Comment (rpreen) [reply test/test_initial.py:55]: something strange is definitely happening, i got notification of the push but it's not showing anywhere

Comment (jim-smith) [reply test/test_initial.py:55]: and the file looks right if you click to open it in codespace on the web page.

**Pull request closed without merging**